docs/docs/blocks/!/All.md

```diff
@@ -0,0 +1,36 @@
+# All
+
+```clojure
+(All
+  :Value [(Any)]
+)
```

Comment:
Is there any way to print those in a clojure like formatting way?
```
(All
  :X [(None)]
  :Value [(Any)])

Reply:
I know might be unnatural but it's kinda standard for clojure/lisp users

Reply:
What is a "clojure like formatting way"?

Reply:
https://github.com/bbatsov/clojure-style-guide

Reply:
Lots of their rules are contradictory, so I don't like that.

Reply:
Well, I'm open to better proposals but I'd rather follow the clojure/edn or even scheme community? How about scheme for example, what's their standard?

Reply:
I don't know what scheme standard is. But what I don't like in the clojure guide is that sometimes they say one space indentation is not good, and in the next section they use a single space and in the next one they use yet again different alignment rules. It should be much simpler: 2 (or 4 spaces) alignment in all cases. It's all about readability and ease on the eyes/brain and should be context-free.

Anyway, that's a topic for another (low-priority) issue. We can skip those concerns in the current PR.

Reply:
Cool agreed